docs_guide/intro/new_doc_project.rst

```diff
@@ -1,6 +1,8 @@
-========================================================
-Create a new documentation project for a Qiskit package.
-========================================================
+==================================
```

Comment:
There's a missing `_` in one of the links in the end of this tutorial. Might have missed it when I wrote it. 

Is it better to fix that in other PR or in this one?

Reply:
Is is the link for yaml? You can suggest changes directly in the review so that I can simply commit your suggestions by a button press:
<img width="534" alt="Screen Shot 2023-01-18 at 12 53 10" src="https://user-images.githubusercontent.com/7631333/213164823-d9027408-b148-43e1-8d41-141c2a607e97.png">

Reply:
Not sure if that works for lines that are far from the changed in this PR.  See https://github.com/github/roadmap/issues/347 for more details.

Reply:
Oh that's right. Actually you can't suggest changes on lines that I didn't change.